omnibus/config/software/erlang-crypto2.rb

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright 2012-2017 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "erlang-crypto2"
+default_version "master"
+
+source git: "https://github.com/jaym/erlang-crypto2.git"
+
+# TODO get Jay to convert this to Apache 2
+license "Reserved"
+license_file "LICENSE"
+
+dependency "erlang"
+dependency "rebar"
+
+build do
+  env = with_standard_compiler_flags(with_embedded_path)
+
+  env['USE_SYSTEM_GECODE'] = "1"
```

Comment:
I have no idea - I can try removing them and seeing if it still compiles. I just copied the erchef definition and left in whatever was there